res/controllers/hid-controller-api.d.ts

```diff
@@ -18,18 +18,54 @@ declare namespace controller {
      * @param dataList Data to send as list of bytes
      * @param length Unused but mandatory argument for backwards compatibility
      * @param reportID 1...255 for HID devices that uses ReportIDs - or 0 for devices, which don't use ReportIDs
-     * @param resendUnchangedReport If set, the report will also be send, if the data are unchanged since last sending [default = false]
+     * @param useNonSkippingFIFO If set, the report will send in FIFO mode
+     *
+     *   FALSE (default):
```

Comment:
lets use the javascript literal name (I guess optionally enclosing it in ``?). This is not COBOL ;)
```suggestion
     *   `false` (default):
```